cumulusci/tasks/salesforce/promote_2gp_package.py

```diff
@@ -0,0 +1,308 @@
+from typing import List, Dict, Optional
+from simple_salesforce.exceptions import SalesforceMalformedRequest
+
+from cumulusci.core.config.util import get_devhub_config
+from cumulusci.core.exceptions import CumulusCIException, TaskOptionsError
+from cumulusci.salesforce_api.utils import get_simple_salesforce_connection
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+
+class Promote2gpPackageVersion(BaseSalesforceApiTask):
+    """
+    Promote a 2GP Package version from a Beta to a Production release.
+
+    The terms "promoted" and "released" are the same in the context of
+    a 2GP package. Once "promoted" a package is considered to be "released".
+    Hence, the "promotion" of a package is just an update to the
+    Package2Version.IsReleased field.
+
+    The abbreviation `spv` in variable names is shorthand for "SubscriberPackageVersion".
+    """
+
+    task_docs = """Promote a Second Generation managed package.
```

Comment:
Unlocked packages also need to be promoted https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_unlocked_pkg_create_pkg_ver_promote.htm